art/attacks/inference/attribute_inference/black_box.py

```diff
@@ -173,3 +173,122 @@ def _check_params(self) -> None:
             raise ValueError("Attack feature must be either an integer or a slice object.")
         if isinstance(self.attack_feature, int) and self.attack_feature < 0:
             raise ValueError("Attack feature index must be positive.")
+
+
+class AttributeInferenceBaseline:
```

Comment:
To be able to identify this attack later as an attribute inference attack it would be good if it inherits from `AttributeInferenceAttack`:
```suggestion
class AttributeInferenceBaseline(AttributeInferenceAttack):
```
The `estimator` can be `None` in the call to `AttributeInferenceAttack.__init(...)__`.